src/main/java/appeng/core/Registration.java

```diff
@@ -91,7 +91,8 @@
 import appeng.capabilities.Capabilities;
 import appeng.core.features.AEFeature;
 import appeng.core.features.registries.P2PTunnelRegistry;
-import appeng.core.features.registries.cell.BasicCellHandler;
+import appeng.fluids.registries.BasicFluidCellHandler;
+import appeng.core.features.registries.cell.BasicItemCellHandler;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Wrong order for 'appeng.core.features.registries.cell.BasicItemCellHandler' import. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=checkstyle%3Acom.puppycrawl.tools.checkstyle.checks.imports.ImportOrderCheck)